packages/examples/src/nestedArrays.ts

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
This example currently doesn't work with the vanilla renderers, since it's not implemented properly. Would it make sense to exlucde it somehow from the vanila examples?